src/types.ts

```diff
@@ -65,6 +65,11 @@ export interface ChangelogOptions extends Partial<ChangelogenOptions> {
    * @default true
    */
   groupByScope?: boolean
+  /**
+   * Use emojis in section titles
+   * @default true
+   */
+  emojis?: boolean
```

Comment:
```suggestion
  emoji?: boolean
```

To describe it as a feature, I think in single form it looks better.